### ISSUE TITLE
Accuracy of Prototree ensemble 5

### ISSUE THREAD

**Issue by Millielele** [M-Nauta/ProtoTree#9]
Thanks for your excellent work!
When i reproduce the Prototree ensemble 5 result, my accuracy is 83, lower than 87.2 in article. Here is my run command: python main_ensemble.py --epochs 100 --log_dir ./runs/protoree_cub --dataset CUB-200-2011 --lr 0.001 --lr_block 0.001 --lr_net 1e-5 --num_features 256 --depth 9 --net resnet50_inat --freeze_epochs 30 --milestones 60,70,80,90,100 --nr_trees_ensemble 5
How can I solve that?

**Comment by M-Nauta** [comment]
Hi Millielele, I am sorry to hear that you cannot reproduce our results. It is difficult to pinpoint the main reason but it could be that the training hyperparameters, random seed, data preprocessing, PyTorch version or other settings were not the same as used in the paper. According to [this answer](https://github.com/M-Nauta/ProtoTree/issues/4#issuecomment-1111632610), training a bit longer could also help.

We have also published a new part-prototype model: [PIP-Net](https://github.com/M-Nauta/PIPNet). Feel free to try it out!
